### PR TITLE
New Garganacl and Naclstack battle sprite + various battle sprite fixes

### PR DESCRIPTION
**Pull request by kittenchilly**
## Description
Replaces the Garganacl and Naclstack sprite with the one from CyanSixFour's repo.

Additionally fixes the following battle sprite issues from #5135
- Garganacl could use a better sprite
- Spidops battle shiny eyes
- Wattrel battle shiny eyes
- Kilowattrel battle shiny legs
- Shroodle battle shiny palette wrong
- Bombirdier battle stray green pixels on wings, shiny toes
- Orthworm battle stray black pixels in mouth
- Veluza battle stray pixels in mouth
- Aerodactyl-Mega battle shiny or normal either doesn't or have a flap at the back of the mouth that should/shouldn't be there.
- Medicham-Mega battle shiny leg shawl thingy loses texture and arm bangles change
- Absol-Mega battle heel spikes not visible
- Salamence-Mega battle eyes could be more triangular
- Lati(a/o)s-Mega battle open mouth looks like a derp tongue
- Diancie-Mega battle shiny outline of waist gold thingy
- Ponyta-Galar battle shiny hair too yellow
- Rapidash-Galar battle shiny hair too yellow
- Magearna battle shiny missing.
- Landorus battle-front shiny tail wrong shade.

## **Discord contact info**
kittenchilly

### CODE REVIEW

File: graphics/pokemon/medicham/mega/back.png
Comment (hedara90) [inline graphics/pokemon/medicham/mega/back.png:1]: Maybe the floaty things should be a bit disconnected from the body 
DSC_0236

**Review by hedara90 — COMMENTED** [review]
Some comments, let me know if you think it should be changed or not

File: graphics/pokemon/medicham/mega/back.png
Comment (kittenchilly) [reply graphics/pokemon/medicham/mega/back.png:1]: I actually changed the sprite to use a new one from the DS sprite project, which is why its pretty different.

File: graphics/pokemon/medicham/mega/front.png
[the remark this answers is not in the document]
Comment (kittenchilly) [reply graphics/pokemon/medicham/mega/front.png:1]: I believe thats intentional, to look more like its model

File: graphics/pokemon/latias/mega/front.png
[the remark this answers is not in the document]
Comment (kittenchilly) [reply graphics/pokemon/latias/mega/front.png:1]: When I was changing this, I looked at both its model and its sprite from the smogon sprite project, which both don't have an open mouth. So I thought it'd be better if it didn't have the mouth visible.

File: graphics/pokemon/medicham/mega/back.png
Comment (kittenchilly) [reply graphics/pokemon/medicham/mega/back.png:1]: Anyways, they are disconnected from the body more than the previous sprite, its kind of hard to make them even more visibly disconnected.

File: graphics/pokemon/latias/mega/front.png
[the remark this answers is not in the document]
Comment (hedara90) [reply graphics/pokemon/latias/mega/front.png:1]: Makes sense

File: graphics/pokemon/garganacl/back.png
[the remark this answers is not in the document]
Comment (kittenchilly) [reply graphics/pokemon/garganacl/back.png:1]: This is the only good backsprite I could find, I don't think I'm good enough to change that sprite that much

File: graphics/pokemon/medicham/mega/front.png
[the remark this answers is not in the document]
Comment (hedara90) [reply graphics/pokemon/medicham/mega/front.png:1]: front
Something like this maybe for the front sprite, then back and front matches a bit better.

[the remark this answers is not in the document]
Comment (kittenchilly) [reply graphics/pokemon/medicham/mega/front.png:1]: Seems good

**Comment by hedara90** [comment]
back
front

**Review by hedara90 — APPROVED** [review]

**Comment by hedara90** [comment]
Checks 18 sprites in #5135

**Pull request merged**